src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java

```diff
@@ -538,6 +538,16 @@ protected boolean handleResponse(BulkItemResponse response,
                   response.getVersion(),
                   response.getIndex()
           );
+
+          log.trace("{} version conflict for operation {} on document '{}' version {}"
```

Comment:
let's have this at info level. otherwise trace logging needs to be enabled.

Reply:
In this case, we can modify warn itself but idea is to switch it off immediately after collecting logs.

Reply:
ok